src/ISM330DLCSensor.cpp

```diff
@@ -418,7 +418,7 @@ ISM330DLCStatusTypeDef ISM330DLCSensor::Get_G_Sensitivity(float *pfData)
     /* Store the sensitivity based on actual full scale. */
     switch( fullScale )
     {
-      case ISM330DLC_ACC_GYRO_FS_G_245dps:
+      case ISM330DLC_ACC_GYRO_FS_G_250dps:
         *pfData = ( float )ISM330DLC_GYRO_SENSITIVITY_FOR_FS_245DPS;
```

Comment:
I would change also the ISM330DLC_GYRO_SENSITIVITY_FOR_FS_245DPS define in ISM330DLC_GYRO_SENSITIVITY_FOR_FS_250DPS